### PR TITLE
Add a new troubleshooting step for Something Went Wrong error

### PR DESCRIPTION
**Pull request by cagriy**
## Proposed change

There is an error case which is not covered in the existing Nest Integration documentation this is when a "Something went wrong, please try again in few minutes." message is displayed. Please note that this error is different from the documented "Something went wrong: Please contact the developer of this app if the issue persists" error.

The addition to the documentation aims to cover this point and offers a work around for one of the known error scenarios.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/nest.markdown
Comment (FletcherAU) [inline source/_integrations/nest.markdown]:
```suggestion
- *Something went wrong, please try again in few minutes.*: According to Google's [Partner Connections Manager Error Reference](https://developers.google.com/nest/device-access/reference/errors/pcm) this error covers all other undocumented internal errors within Partner Connections. One of the issues that causes this error is synchronisation problems between the Nest and Google Home apps. Confirm that your Nest device is visible within both apps under the same Home. If it is missing within Google Home, create a new dummy home on the Nest app as this triggers the synchronisation process. (This is the workaround recommended by the Google support team). The dummy entry can be deleted once the Nest device is visible within the Google Home app.
```
Slight reword suggestion

Comment (cagriy) [reply source/_integrations/nest.markdown]: Thank you for the tidy-up @FletcherAU

Comment (frenck) [reply source/_integrations/nest.markdown]: If you are agreeing with that suggestion @cagriy, why don't you accept/merge it?

Comment (cagriy) [reply source/_integrations/nest.markdown]: Hi @frenck , on my end the PR still seems to be waiting for approval.

Comment (frenck) [reply source/_integrations/nest.markdown]: No, there is a review comment above, including a suggestion.

As you haven't changed anything at this point, it is awaiting your action. Feel free to adjust the PR to the suggestion or commit the suggestion directly.

Marking this PR draft until is has been resolved.

../Frenck

Comment (cagriy) [reply source/_integrations/nest.markdown]: I am sorry, I reverted the draft by mistake, would you mind setting it as draft again?

Comment (frenck) [reply source/_integrations/nest.markdown]: >  I reverted the draft by mistake, would you mind setting it as draft again?

But you made the change? You can draft it yourself if needed. Do you plan on making more changes?

**Comment by cagriy** [comment]
@FletcherAU 's changes committed

File: source/_integrations/nest.markdown
Comment (cagriy) [reply source/_integrations/nest.markdown]: @frenck , would you mind checking if it is now good for approval please ?

Comment (frenck) [reply source/_integrations/nest.markdown]: I think you are not aware of how the GitHub PR process works. You mark it ready when you think it is good for approval (that is now our decision to make). After that, a review will happen, by "someone". You have to await that.

Comment (FletcherAU) [reply source/_integrations/nest.markdown]: @cagriy You may find this [guide](https://developers.home-assistant.io/docs/review-process) handy, it's got a pretty good summary of the review process. If you have questions regarding the process after reading it/in the future the HA Discord is also a good place to ask :)

File: source/_integrations/nest.markdown
Comment (frenck) [inline source/_integrations/nest.markdown]:
```suggestion
- *Something went wrong, please try again in a few minutes*: According to Google's [Partner Connections Manager Error Reference](https://developers.google.com/nest/device-access/reference/errors/pcm), this error covers all other undocumented internal errors within Partner Connections. One of the issues that cause this error is synchronization problems between the Nest and Google Home apps. Confirm that your Nest device is visible within both apps under the same Home. If it is missing within Google Home, create a new dummy home on the Nest app, which triggers the synchronization process. (This is the workaround recommended by the Google support team). The dummy entry can be deleted once the Nest device is visible within the Google Home app.
```

**Review by frenck — APPROVED** [review]
Thanks, @cagriy 👍 

../Frenck

**Pull request merged**
